docs/README.md

```diff
@@ -13,8 +13,18 @@ This is the documentation of AMS, it's focused on the **0.10.x version.**
 ## How to
 
 - [How to add root key](howto/add_root_key.md)
-- [How to add pagination links](howto/add_pagination_links.md)
 - [Using AMS Outside Of Controllers](howto/outside_controller_use.md)
+- [How to add pagination links](howto/add_pagination_links.md)
+
+#### JSON API Resource `type`
+
+##### Implicit
+
+Resource class name is used, e.g. the type of a `Post` will be `posts` in the resulting json api resource.
+
+##### Explicitly
+
+In the Serializer, declare `type foo`, and `"type": "foo"` will be present in the corresponding resulting json api resource.
```

Comment:
This makes me think we really need a page that describes how  we build (and configure) different parts of a JSON API document.  This would be under Resource. 

And also that we might want to specialize a JSON API Resource vs. JSON Resource (which what I'd like to rename AM::S to), like in https://github.com/cerebris/jsonapi-resources